apps/openmw/mwgui/savegamedialog.cpp

```diff
@@ -361,6 +378,11 @@ namespace MWGui
             << MWBase::Environment::get().getWorld()->getMonthName(mCurrentSlot->mProfile.mInGameTime.mMonth)
             <<  " " << hour << " " << (pm ? "#{sSaveMenuHelp05}" : "#{sSaveMenuHelp04}");
 
+        if (Settings::Manager::getBool("timeplayed","Saves"))
+        {
+            text << "\n" << "Time played: " << formatTimeplayed((int)floor(mCurrentSlot->mProfile.mTimePlayed));
```

Comment:
It makes it easier to do the math... Cleaner code to pass the format function a long int than a double... Because then you'll get into using fmods and floors anyway